### ISSUE TITLE
Not working in Foundry v9

### ISSUE THREAD

**Issue by dczwardCZAR** [BlitzKraig/fvtt-EasyTable#18]
It doesn't seem to be compatible with Foundry v9. I'm posting here to get attention to the issue.

**Comment by brunocalado** [comment]
Hi.

Please remove the foundry vtt warning.

Replace "compatibleCoreVersion":"0.9.xxx" by "compatibleCoreVersion":"9".

Docs: https://gitlab.com/foundrynet/foundryvtt/-/issues/5635

**Comment by Shuggaloaf** [comment]
>Replace "compatibleCoreVersion":"0.9.xxx" by "compatibleCoreVersion":"9".

Current release `compatibleCoreVersion` is `0.8.6`. Did you mean replace that with `9`? If not, where is the `0.9.xxx` you are referring to?

**Comment by dczwardCZAR** [comment]
Yes, I think that's what he means. I edited the v1.3.3 EasyTable module.json file, line 22 to read...
`  "compatibleCoreVersion": "9",`
... and it seems to be working.

**Comment by Shuggaloaf** [comment]
The warning goes away but there are actual features not working with v9. Exporting tables for example. 

Is this mod no longer under active development?

**Comment by brunocalado** [comment]
> The warning goes away but there are actual features not working with v9. Exporting tables for example.
> 
> Is this mod no longer under active development?

Worked in v9. Maybe ask in the league. This module is very useful.

Invite URL for this server: https://discord.gg/fvttdevleague

**Comment by Shuggaloaf** [comment]
@brunocalado Just to be clear - are you saying BRT exporting is working for you in the current version of v9 or one of the earlier v9 releases?

**Comment by brunocalado** [comment]
> @brunocalado Just to be clear - are you saying BRT exporting is working for you in the current version of v9 or one of the earlier v9 releases?

yes
![image](https://user-images.githubusercontent.com/4999783/172203339-9b44fad5-131a-435c-b8fa-6eb950327745.png)

**Comment by Shuggaloaf** [comment]
Yeah import works just not export. I'll check with the league tho as it happens even with no other mods enabled.
